pkg/controller/node/node_controller.go

```diff
@@ -460,6 +438,11 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling Node")
 
+	if r.sharedInfo.AddNodeTag == false {
```

Comment:
Check this flag in line 72 or 78 when init node controller or start watching nodes

Reply:
Good idea, done.